src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -699,6 +689,9 @@ namespace AirLoopHVACDOAS {
                                                state.dataAirLoop->OutsideAirSys(thisDOAS.m_OASystemNum).ComponentType(CompNum)));
                         errorsFound = true;
                     }
+                    if (CoolingCoilOrder > FanOrder) {
+                        thisDOAS.FanHeatAdded = true;
+                    }
```

Comment:
When a fan is placed before a cooling coil, FanHeat should be added in cooling coil capacity size

Reply:
You don't even need FanBeforeCoolingCoilFlag do you. If CoolingCoilOrder > FanOrder then it is blow through.

Reply:
What we have no is a single enum for fan placement.

    enum class FanPlacement
    {
        Invalid = -1,
        BlowThru,
        DrawThru,
        Num
    };

This fan location applies to both the cooling and heating coil. But the fan could actually be in between the coils. That can not happen now but could in the future. It would be better to have enums for BlowThruCooling and BlowThruHeating, and same for DrawThru, but we just would not use the heating enums yet. Which means we would not define the heating enums yet. Could you change FanBeforeCoolingCoilFlag to be DataAirSystems::FanPlacement::supFanLocation and use existing enums? That way we can expand on this in the future.

Reply:
@rraustad Phase 1 allows a supply fan can be in any order. In other words, BlowThru and DrawThru restrictions are removed in this module. Phase 2 is implemented based on Phase 1. Therefore, it is not appropriate to use existing FanPlacement enum. Phase 2 checks locations of both fan and cooling coil. If a fan is placed before a cooling coil, the fan heat will be added in cooling capacity sizing.